sensor_stack/sparton/src/sparton.py

```diff
@@ -241,10 +246,71 @@ def getData():
     allData.data[12] = temp
     
     return allData
-    
- 
-if __name__ == '__main__':
-    """    
+
+def getOrientation():
+
+    global roll
+    global pitch
+    global yaw
+
+    quaternion = 
+
+    return quaternion
+
+def getOrientationCovariance():
+
+    # global roll
+    # global pitch
+    # global yaw
+
+    cov_mat = [0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0]
+
+    return cov_mat
+
+def getAngularVelocityCovariance():
+
+    # global gx
+    # global gy
+    # global gz
+
+    cov_mat = [0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0]
+
+    return cov_mat
+
+
+def getLinearAccelerationCovariance():
+
+    # global ax
+    # global ay
+    # global az
+
+    cov_mat = [0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0]
+
+    return cov_mat
+
+def getQuaternion():
```

Comment:
@athityakumar Appreciate your effort in writing this function, but isnt the RPY -> Quarternion conversion a part of the Python library? I would use that function (Either from ROSpy or from the Python standard lib)

Check the code of the python pose estimator to find this function, I remember using it but don't remember what the name was, or which import you need to make.

cc @pranaypratyush and @aniket11bh Please point him to the right function if you know of it.